app/controllers/transformation-controller.js

```diff
@@ -244,7 +244,7 @@ function _respond(res, survey) {
  * @return { string } - a hash
  */
 function _getCombinedHash(survey) {
-    const FORCE_UPDATE = 1;
```

Comment:
Some context for this change: when testing previously cached forms, I noticed that images were failing to load because they had been cached with `data-offline-src` attributes. This forces a retransformation, which probably isn't necessary if we add some migration logic to the client.

Reply:
This is no longer necessary at all after 8449758, it'll be removed shortly.